pingcap/tidb
# *: fix add index metrics

Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>



### What problem does this PR solve?


Issue Number: close #31115

Problem Summary:
Adding index metrics don't show the rate of adding index only, it combines with worker duration.

### What is changed and how it works?

1. Fix adding index metrics, maybe need to add worker duration in another PR.
2. Change 'speed' to 'rate' because 'speed' is a vector and 'rate' is a scalar.

### Check List

Tests 

1. start a cluster
2. check the metrics

Side effects


Documentation


### Release note



```release-note
None
```
